### PR TITLE
One fix per day 10

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by Arsnael** [comment]
```
[fecd9726c04f92a754e5823aabc14ccaee75323a] [INFO] There is 1 error reported by Checkstyle 8.29 with checkstyle.xml ruleset.
[fecd9726c04f92a754e5823aabc14ccaee75323a] [ERROR] src/main/java/org/apache/james/mailbox/store/mail/model/impl/PropertyBuilder.java:[44,8] (imports) UnusedImports: Unused import - java.util.Iterator.
```

**Comment by Arsnael** [comment]
```
[249217e5489d5a29129fbd9d10cc2992b67f6899] [INFO] There is 1 error reported by Checkstyle 8.29 with checkstyle.xml ruleset.
[249217e5489d5a29129fbd9d10cc2992b67f6899] [ERROR] src/main/java/org/apache/james/imap/processor/IdleProcessor.java:[39,8] (imports) UnusedImports: Unused import - org.apache.james.imap.api.process.ImapLineHandler.
```

File: mailbox/plugin/deleted-messages-vault/src/test/java/org/apache/james/vault/DeletedMessageZipperTest.java [hunk 1]
```diff
@@ -182,13 +182,8 @@ void zipShouldCloseParameterOutputStreamWhenGettingException() throws Exception
...
             doThrow(new IOException("mocked exception")).when(zipper).createEntry(any(), any());
-            DeletedMessageContentLoader contentLoader = spy(new DeletedMessageContentLoader() {
-                // lambdas are final and thus can't be spied
-                @Override
-                public Optional<InputStream> load(DeletedMessage deletedMessage) {
-                    return Optional.of(new ByteArrayInputStream(CONTENT));
-                }
-            });
+            // lambdas are final and thus can't be spied
+            DeletedMessageContentLoader contentLoader = spy(deletedMessage -> Optional.of(new ByteArrayInputStream(CONTENT)));
```
Comment (remk) [inline mailbox/plugin/deleted-messages-vault/src/test/java/org/apache/james/vault/DeletedMessageZipperTest.java]: I think that this change is in  contradiction with the comment no ?

**Comment by Arsnael** [comment]
output log lost

test this please

File: server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/SieveMailAdapter.java [hunk 2]
```diff
@@ -383,7 +385,7 @@ public boolean isInBodyText(List<String> phrasesCaseInsensitive) throws SieveMai
                 .contentTypes(Lists.newArrayList("text/plain"))
                 .includeHeaders(false)
                 .caseInsensitive(false)
-                .searchContents(Lists.transform(phrasesCaseInsensitive, s -> s)).build()
+                .searchContents(phrasesCaseInsensitive.stream().map(s -> s).collect(Guavate.toImmutableList())).build()
```
Comment (Arsnael) [inline server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/SieveMailAdapter.java]: Is the map necessary here?

File: server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/VacationAction.java [hunk 3]
```diff
@@ -80,10 +81,10 @@ private boolean isStillInVacation(ActionVacation actionVacation, int dayDifferen
...
     private boolean isValidForReply(final Mail mail, ActionVacation actionVacation, final ActionContext context) {
         Set<MailAddress> currentMailAddresses = ImmutableSet.copyOf(mail.getRecipients());
-        Set<MailAddress> allowedMailAddresses = ImmutableSet.<MailAddress>builder().addAll(
-            Lists.transform(actionVacation.getAddresses(), s -> retrieveAddressFromString(s, context)))
-            .add(context.getRecipient())
-            .build();
+        Set<MailAddress> allowedMailAddresses = Stream.concat(
+            actionVacation.getAddresses().stream().map(s -> retrieveAddressFromString(s, context)),
+            Stream.of(context.getRecipient()))
+            .collect(Guavate.toImmutableSet());
```
Comment (Arsnael) [inline server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/VacationAction.java]: line breaks and indents in this block

File: server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/SieveMailAdapter.java (hunk 2 above)
Comment (mbaechler) [reply server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/jsieve/SieveMailAdapter.java]: there's a generic mismatch problem but call ImmutableList.copy does the trick

**Comment by Arsnael** [comment]
Merged

**Pull request closed without merging**